changelog/8761.feature.rst

```diff
@@ -1 +1 @@
-New :ref:`version-tuple` attribute, which makes it simpler for users to do something depending on the pytest version (such as declaring hooks which are introduced in later versions).
+New :ref:`version_tuple` attribute, which makes it simpler for users to do something depending on the pytest version (such as declaring hooks which are introduced in later versions).
```

Comment:
I'm not sure which one is correct here - @nicoddemus @RonnyPfannschmidt?

Reply:
Ah, it looks like after resolving the previous build failure, Sphinx now fails with:

```
/home/runner/work/pytest/pytest/doc/en/_changelog_towncrier_draft.rst:114: WARNING: undefined label: version_tuple
```

here, so I was wrong and `version-tuple` was correct here. I thought it was referring to the variable `version_tuple`, but it actually refers to [a `version-tuple` label](https://github.com/pytest-dev/pytest/commit/7eb0792cba262ef2e659525e2e9f93742169efab#diff-039da146358bc54358e2b627fe9544ac13412c5f4bcc913277b45ac5256e8c61R239) added to the docs.